src/main/java/com/amazonaws/encryptionsdk/kms/KmsMasterKey.java

```diff
@@ -1,11 +1,11 @@
 /*
```

Comment:
This file was a revert to mainline while adding the new deprecated messaging and updating the new of a few variables.